agent/api/container/container.go

```diff
@@ -115,6 +115,8 @@ type Container struct {
 	Memory uint
 	// Links contains a list of containers to link, corresponding to docker option: --link
 	Links []string
+	//
+	LogRouter *LogRouter `json:"logRouter"`
```

Comment:
can you add a unit test in https://github.com/aws/amazon-ecs-agent/blob/master/agent/acs/handler/payload_handler_test.go to make sure the new field can be populated from acs payload? also this field is missing comment here

Reply:
added.